src/platform-includes/enriching-events/attach-viewhierarchy/android.mdx

```diff
@@ -3,3 +3,25 @@
   <meta-data android:name="io.sentry.attach-view-hierarchy" android:value="true" />
 </application>
 ```
+
+### Jetpack Compose Support using the Sentry Kotlin Compiler Plugin
+
+<Note>
+
+The Sentry Kotlin Compiler plugin is considered _experimental_. Give it a try and provide early feedback [on github](https://github.com/getsentry/sentry-android-gradle-plugin).
+
+</Note>
+
+If you application is using Jetpack Compose and you want to see the function names of your `@Composable` elements, you can apply the Sentry Kotlin Compiler Plugin. The plugin needs to be applied to all modules which contain `@Composable` elements.
+
+```groovy {filename:app/build.gradle}
+plugins {
+  id "io.sentry.kotlin.compiler.gradle" version "{{@inject packages.version('sentry.kotlin.compiler-plugin', '3.0.0') }}"
+}
+```
+
+```kotlin {filename:app/build.gradle.kts}
+plugins {
+  id("io.sentry.kotlin.compiler.gradle") version "{{@inject packages.version('sentry.kotlin.compiler-plugin', '3.0.0') }}"
+}
+```
```

Comment:
I guess having a screenshot here showcasing compose view hierarchy would be nice. Maybe we could have a screenshot per platform (I think now it's under common)

Reply:
@markushi can we still do this? Kinda a bummer to have ios view hierarchy on the android docs 🙈 
![image](https://github.com/getsentry/sentry-docs/assets/4999776/4282937c-e759-4d9b-ae4d-a3bc39ede2ef)

I think we'd have to add `<PlatformSection notSupported="..."/>` to the common content here https://github.com/getsentry/sentry-docs/blob/a30b28d50d92b32db81d623f7fd663ba21dcab5e/src/platforms/common/enriching-events/viewhierarchy.mdx?plain=1#L45 and then add our own section with a proper screenshot. Not sure if it's supported to have images per platform.